docs/generate_recipe_doc.py

```diff
@@ -0,0 +1,156 @@
+"""
+Helper script to autogenerate documentation for recipes.
+"""
+
+import os
+import sys
+import json
+import graphviz
+
+# These are the first strings that will be generated in the file.
+HEADER = """
+# Recipe list
+
+This is an auto-generated list of dfTimewolf recipes.
+
+To regenerate this list, from the repository root, run:
+
+```
+pipenv install --dev
+python docs/generate_recipe_doc.py data/recipes
+```
+
+---
+"""
+
+# This is the template for representing a recipe's documentation
```

Comment:
Comments left in for a reason?

Reply:
I guess this was just to show how the template is intended to be used. I can remove it it you find it confusing.